tekton/pipeline_run_test.go

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2022 Red Hat Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tekton
+
+import (
+	"context"
+	"reflect"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/redhat-appstudio/release-service/api/v1alpha1"
+	tektonv1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ExtraParams struct {
+	Name  string
+	Value tektonv1beta1.ArrayOrString
+}
+
+type ReleasePipelineParams struct {
+	Prefix    string
+	Namespace string
+	Extra     ExtraParams
+}
+
+var _ = Describe("PipelineRun", func() {
+
+	var params *ReleasePipelineParams
+	var releasePipelineRun *ReleasePipelineRun
+
+	var release *v1alpha1.Release
+	var releaseParams *v1alpha1.ReleaseSpec
+
+	BeforeEach(func() {
+
+		params = &ReleasePipelineParams{
+			Prefix:    "mypipeline",
```

Comment:
This is only used once in the file. I don't necessarily dislike it as a variable at the top, but I think we should be consistent. There are plenty of other strings also used once that aren't variables in the params

Reply:
You are right. This is because I was creating `PipelineRuns` for each test, than later I moved it to top and indeed, this does not need to be a variable anymore.